### PR TITLE
[翻译更新]permalinks and pagination

### PR DESCRIPTION
**Pull request by TimoTokki**
<dl>
<dt>
permalinks
</dt>
<dd>
added new content and modified some translations
</dd>
<dt>
pagination
</dt>
<dd>
added new content and modified some translations
</dd>
</dl>

### CODE REVIEW

File: site/_docs/permalinks.md [hunk 1]
```diff
@@ -121,16 +180,26 @@ Jekyll 支持以灵活的方式管理你网站的链接，你可以通过 [Confi
...
 
 比如文件名： `/2009-04-29-slap-chop.textile`
 
 <div class="mobile-side-scroller">
 <table>
   <thead>
     <tr>
-      <th>设置</th>
-      <th>对应的 URL </th>
+      <th>URL　模板</th>
```
Comment (xcatliu) [inline site/_docs/permalinks.md]: 这里是全角空格？

File: site/_docs/permalinks.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 layout: docs
 title: 永久链接
 permalink: /docs/permalinks/
-translators: debbbbie
+translators: [debbbbie,TimoTokki]
```
Comment (xcatliu) [inline site/_docs/permalinks.md]: 逗号后最好有空格

**Comment by TimoTokki** [comment]
已更改

**Comment by xcatliu** [comment]
Published, thanks!

**Pull request merged**